src/main/resources/org/csanchez/jenkins/plugins/kubernetes/Messages.properties

```diff
@@ -9,3 +9,10 @@ KubernetesSlave.HomeWarning=[WARNING] HOME is set to / in the agent container. Y
     entry in /etc/passwd. Please add a user to your Dockerfile or set the HOME environment \
     variable to a valid directory in the pod template definition.
 KubernetesCloudNotAllowed.Description=Kubernetes cloud {0} is not allowed for folder containing job {1}
+KubernetesCloud.skipTlsVerifyNotAllowedInFIPSMode=Skipping TLS verification is not allowed in FIPS mode
+KubernetesCloud.serverCertificateIsNotApprovedInFIPSMode=Certificate is not valid: {0}
+KubernetesCloud.serverCertificateKeySize=Invalid key size, at least 2048 is needed
+KubernetesCloud.serverCertificateKeySizeEC=Invalid curve size, at least 224 is needed
```

Comment:
**question:** In FIPS mode only right? Shouldn't the error message add this precision?